src/reducers/tags.js

```diff
@@ -77,6 +78,16 @@ export const tagsByCommunity = (state = {}, action) => {
     case FETCH_CURRENT_USER:
     case CREATE_COMMUNITY:
       return mergeLeftNavTags(state, get('left_nav_tags', payload))
+    case UPDATE_POST_EDITOR:
+      const { tagDescriptions } = payload
+      let { slug } = meta
+      if (isEmpty(tagDescriptions)) return state
+      let tags = Object.keys(tagDescriptions).map(key => ({...tagDescriptions[key], name: key, followed: true}))
+      return {...state, [slug]: mergeList(state[slug], tags, 'name')}
```

Comment:
waiting until save makes way more sense.

Don't need to update locally for communities other than the current one, because if the user goes to a different community there will be a fetchLeftNavTags that updates the store.